simpleclient_hotspot/src/main/java/io/prometheus/client/hotspot/ThreadExports.java

```diff
@@ -73,8 +77,42 @@ void addThreadMetrics(List<MetricFamilySamples> sampleFamilies) {
         "jvm_threads_deadlocked_monitor",
         "Cycles of JVM-threads that are in deadlock waiting to acquire object monitors",
         nullSafeArrayLength(threadBean.findMonitorDeadlockedThreads())));
+
+    GaugeMetricFamily threadStateFamily = new GaugeMetricFamily(
+      "jvm_threads_state",
+      "Current count of threads by state",
+      Collections.singletonList("state"));
+
+    Map<Thread.State, Integer> threadStateCounts = getThreadStateCountMap();
+    for (Thread.State state : Thread.State.values()) {
+      String stateName = state.toString().toLowerCase();
```

Comment:
Why lowercase?

Reply:
Doesn't make sense anymore. Before those names where used in the metrics name, but since that changed, I can just use the normal name - without lower case.